code/modules/mob/living/silicon/say.dm

```diff
@@ -10,24 +10,41 @@
 	if(issilicon(src))
 		var/mob/living/silicon/S = src
 		desig = trim_left(S.designation + " " + S.job)
-	var/message_a = say_quote(message)
-	var/rendered = "Robotic Talk, <span class='name'>[name]</span> <span class='message'>[message_a]</span>"
+	var/large_message_a = say_quote(message, list("robot big"))
+	var/message_a = say_quote(message, list("robot"))
```

Comment:
```suggestion
	var/large_message_a = say_quote(message, list("robot big"))
	var/message_a = say_quote(message, list("robot"))
	var/mob/living/silicon/ai/true_ai_core
	if(iscyborg(src))  // this detects if a borg is AI shell, so that they can be loud always
		var/mob/living/silicon/robot/ai_shell = src
		true_ai_core = ai_shell.mainframe
```